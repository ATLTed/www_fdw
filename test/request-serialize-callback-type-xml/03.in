select * from www_fdw_test where title='TITLE' and link='LINK' or not substring(snippet,2,5)='NIPPE';
