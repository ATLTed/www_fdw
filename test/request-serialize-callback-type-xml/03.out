INFO:  quals parameter: <node name="BoolExpr"><params><param name="boolop" value="OR"/><param name="location" value="-1"/></params><children><node name="BoolExpr"><params><param name="boolop" value="AND"/><param name="location" value="-1"/></params><children><node name="OpExpr"><params><param name="opno" value="98"/><param name="opfuncid" value="67"/><param name="opresulttype" value="16"/><param name="opretset" value="f"/><param name="opcollid" value="0"/><param name="inputcollid" value="100"/><param name="location" value="38"/></params><children><node name="Var" value=""><params><param name="varno" value="1"/><param name="varattno" value="1"/><param name="vartype" value="25"/><param name="vartypmod" value="-1"/><param name="varcollid" value="100"/><param name="varlevelsup" value="0"/><param name="varnoold" value="1"/><param name="varoattno" value="1"/><param name="location" value="33"/></params></node><node name="Const" value="TITLE"><params></params></node></children></node><node name="OpExpr"><params><param name="opno" value="98"/><param name="opfuncid" value="67"/><param name="opresulttype" value="16"/><param name="opretset" value="f"/><param name="opcollid" value="0"/><param name="inputcollid" value="100"/><param name="location" value="55"/></params><children><node name="Var" value=""><params><param name="varno" value="1"/><param name="varattno" value="2"/><param name="vartype" value="25"/><param name="vartypmod" value="-1"/><param name="varcollid" value="100"/><param name="varlevelsup" value="0"/><param name="varnoold" value="1"/><param name="varoattno" value="2"/><param name="location" value="51"/></params></node><node name="Const" value="LINK"><params></params></node></children></node></children></node><node name="OpExpr"><params><param name="opno" value="531"/><param name="opfuncid" value="157"/><param name="opresulttype" value="16"/><param name="opretset" value="f"/><param name="opcollid" value="0"/><param name="inputcollid" value="100"/><param name="location" value="92"/></params><children><node name="FuncExpr"><params><param name="funcid" value="936"/><param name="funcresulttype" value="25"/><param name="funcretset" value="f"/><param name="funccollid" value="100"/><param name="inputcollid" value="100"/><param name="location" value="70"/></params><children><node name="Var" value=""><params><param name="varno" value="1"/><param name="varattno" value="3"/><param name="vartype" value="25"/><param name="vartypmod" value="-1"/><param name="varcollid" value="100"/><param name="varlevelsup" value="0"/><param name="varnoold" value="1"/><param name="varoattno" value="3"/><param name="location" value="80"/></params></node><node name="Const" value="2"><params></params></node><node name="Const" value="5"><params></params></node></children></node><node name="Const" value="NIPPE"><params></params></node></children></node></children></node>
